<launch>

   <arg name="rgb_topic" default="/camera/image_raw"/>
   <arg name="camera_info_topic" default="/camera/camera_info"/>
   <arg name="pose_topic" default="/pose"/>
   <arg name="ball_diameter" default="0.11"/>

   <!-- base_link -> camera_link transform -->
   <node pkg="tf" type="static_transform_publisher" name="base2camera_link_tf_publisher" args="-0.28 0 0.6 0 0 0 /base_link /camera_link 100"/>
   <node pkg="drone_detector_tracking" type="dji_tf.py" name="dji_tf_publisher" output="screen"/>

   <node pkg="drone_detector_tracking" type="color_detector_node" name="color_detector" output="screen">
      <param name="rgb_img_topic" type="str" value="$(arg rgb_topic)"/>
      <param name="camera_info_topic" type="str" value="$(arg camera_info_topic)"/>
      <param name="pose_topic" type="str" value="$(arg pose_topic)"/>
      <param name="ball_diameter" type="double" value="$(arg ball_diameter)"/>
   </node>


</launch>
